src/backend/PortabilityService.DependencyInjection/InjectConfiguration.cs

```diff
@@ -0,0 +1,65 @@
+﻿// MIT License
```

Comment:
this looks like a duplicate of `src/backend/Functions/FunctionsDIConfiguration.cs` Can we consolidate this?

Reply:
Ideally I'd like to have project specific configuration, instead of this one in the class library.  However, Functions runtime doesn't load that extension in `FunctionsDIConfiguration`, so I might have to remove that file at this moment, and investigate the issue further separately.

---
In reply to: [178095468](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178095468) [](ancestors = 178095468)

Reply:
Removed `FunctionsDIConfiguration` for now

---
In reply to: [179244852](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r179244852) [](ancestors = 179244852,178095468)